content/intro-to-storybook/react/zh-CN/get-started.md

```diff
@@ -1,57 +1,55 @@
 ---
-title: '开始吧'
-tocTitle: '从头开始'
-description: '在你的开发环境下, 设置 React Storybook '
+title: 'Storybook React 教程'
+tocTitle: '开始吧'
+description: '在你的开发环境下设置 Storybook'
 commit: '8741257'
 ---
 
-<div class="aside">
-<p>
-  此翻译未更新！您可以通过单击页面底部的链接来帮助我们改进它，不仅团队会感激它，而且整个社区都会感激不尽
-</p>
-</div>
-
-Storybook 是在开发模式下 与 您的应用程序一起运行的. 它可以帮助您构建 UI 组件,并与 应用程序的 业务逻辑和上下文 隔离开来. 本期"学习 Storybook"适用于 **React**; `Vue和Angular`版本即将推出.
+Storybook 在开发环境下与你的程序一起运行。它可以帮助你隔离业务逻辑和应用程序上下文独立构建 UI 组件。此版本的 Storybook 教程适用于 React。其他版本还有：[React Native](/react-native/en/get-started)、[Vue](/vue/en/get-started)、[Angular](/angular/en/get-started) 以及 [Svelte](/svelte/en/get-started)。
 
 ![Storybook and your app](/intro-to-storybook/storybook-relationship.jpg)
 
-> 整个页面 -> 拿出各种组件 -> 分隔出 **每个组件** /`组件组合`用来测试与文档说明
-
 ## 设置 React Storybook
 
-我们需要按照几个步骤设置 Storybook 环境. 首先,我们想要使用[Create React App](https://github.com/facebook/create-react-app) (**CRA**) 快速设置我们的环境,并启用[Storybook](https://storybook.js.org/)和[ jest ](https://facebook.github.io/jest/)测试我们创建的应用. 让我们运行以下命令:
+我们将需要通过几个步骤设置环境。首先，我们需要使用[Create React App](https://github.com/facebook/create-react-app) (CRA) 来设置我们的构建系统，并开启 [Storybook](https://storybook.js.org/) 以及 [Jest](https://facebook.github.io/jest/) 在我们创建的应用程序中进行测试。 让我们运行以下命令：
 
 ```bash
-# 创建应用:
+# 创建应用程序
 npx create-react-app taskbox
+
 cd taskbox
 
-# 加入 Storybook:
-npx -p @storybook/cli getstorybook
+# 添加 Storybook
+npx -p @storybook/cli sb init
 ```
```

Comment:
@STPace i know that the comments in the code snippets used in this version are translated, but that's something that we're shifting away from. We're aware that it brings the translation "closer to home" (pardon the bad pun), but it will introduce a bit of an overhead when proofreading and maintaining. If you don't mind revert them back to english when you're able.

Reply:
Okay i have pushed 😺